.github/workflows/run.yml

```diff
@@ -0,0 +1,18 @@
+
+name: Run lintlizard on project repository
```

Comment:
oh I did see the file in `.github/workflows` directory but given the name of the workflow file and that no GHA was triggered for push/pull-requests, I assumed it might be a workflow that publishes a package to pypi.

I'll remove this.